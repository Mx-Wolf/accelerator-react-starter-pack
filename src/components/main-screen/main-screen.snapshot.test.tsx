import * as ShallowRenderer from 'react-test-renderer/shallow';
import MainScreen from './main-screen';

describe('Component: MainScreen', () => {
  it('should render correctly', () => {
    const view = ShallowRenderer.createRenderer();
    view.render(<MainScreen />);
    expect(view).toMatchSnapshot();
  });
});
