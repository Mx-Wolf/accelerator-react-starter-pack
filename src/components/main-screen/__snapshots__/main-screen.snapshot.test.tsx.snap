// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: MainScreen should render correctly 1`] = `
ReactShallowRenderer {
  "_context": Object {},
  "_didScheduleRenderPhaseUpdate": false,
  "_dispatcher": Object {
    "readContext": [Function],
    "useCallback": [Function],
    "useContext": [Function],
    "useDebugValue": [Function],
    "useDeferredValue": [Function],
    "useEffect": [Function],
    "useImperativeHandle": [Function],
    "useLayoutEffect": [Function],
    "useMemo": [Function],
    "useReducer": [Function],
    "useRef": [Function],
    "useResponder": [Function],
    "useState": [Function],
    "useTransition": [Function],
  },
  "_element": <MainScreen />,
  "_firstWorkInProgressHook": null,
  "_forcedUpdate": false,
  "_instance": null,
  "_isReRender": false,
  "_newState": null,
  "_numberOfReRenders": 0,
  "_renderPhaseUpdates": null,
  "_rendered": <React.Fragment>
    <Icons />
    <div
      className="wrapper"
    >
      <Header />
      <main
        className="page-content"
      >
        <div
          className="container"
        >
          <h1
            className="page-content__title title title--bigger"
          >
            Каталог гитар
          </h1>
          <ul
            className="breadcrumbs page-content__breadcrumbs"
          >
            <li
              className="breadcrumbs__item"
            >
              <Link
                className="link"
                to="#"
              >
                Главная
              </Link>
            </li>
            <li
              className="breadcrumbs__item"
            >
              <Link
                className="link"
                to="#"
              >
                Каталог
              </Link>
            </li>
          </ul>
          <Catalog />
        </div>
      </main>
      <Footer />
    </div>
  </React.Fragment>,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": Array [],
    "_renderer": [Circular],
  },
  "_workInProgressHook": null,
}
`;
